src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseContentProvider.java

```diff
@@ -84,9 +85,8 @@ public DefendantResponseContentProvider(
                 );
                 break;
             case FULL_ADMISSION:
-                content.putAll(
-                    fullAdmissionResponseContentProvider.createContent((FullAdmissionResponse) defendantResponse)
-                );
+                fullAdmissionResponseContentProvider.createContent((FullAdmissionResponse) defendantResponse,
+                    claim.getTotalAmountTillToday().orElse(BigDecimal.ZERO));
```

Comment:
When total claim amount till to day be a valid as ZERO?

Reply:
@sabahirfan this will never occur. This is just if null to get it to ZERO.

Reply:
If expectation is that it should never occur, then better option would be to throw Exception.

Reply:
@sabahirfan this is addressed.